src/Maestro/Maestro.Web/Api/v2019_01_16/Controllers/BuildsController.cs

```diff
@@ -0,0 +1,173 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Maestro.Data;
+using Maestro.Web.Api.v2019_01_16.Models;
+using Microsoft.AspNetCore.ApiPagination;
+using Microsoft.AspNetCore.ApiVersioning;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Swashbuckle.AspNetCore.Annotations;
+using Build = Maestro.Data.Models.Build;
+
+namespace Maestro.Web.Api.v2019_01_16.Controllers
+{
+    public class BuildsController_ApiRemoved : Maestro.Web.Api.v2018_07_16.Controllers.BuildsController
+    {
+        public BuildsController_ApiRemoved(BuildAssetRegistryContext context)
+            : base(context)
+        {
+        }
+
+        [ApiRemoved]
+        public override sealed IActionResult GetAllBuilds(
```

Comment:
Technically for most of these you don't need to "ApiRemoved" them. If the new action method has the same parameters and return type as the old method you can just override it and write the new implementation.

You also don't need a separate class to contain the [ApiRemoved] things.

Reply:
Thats odd, you should just be able to override and change the [SwaggerResponse] attribute, it shouldn't need the [HttpGet] attribute again.